### PR TITLE
Update New-CsGroupPolicyAssignment.md

### PR DESCRIPTION
**Pull request by JSHooper**
Change to emphasise that the assignment is only to direct members of the group via a !Note.

Regards

### CODE REVIEW

File: teams/teams-ps/teams/New-CsGroupPolicyAssignment.md [hunk 1]
```diff
@@ -27,7 +27,10 @@ The group policy assignment rank is set at the time a policy is being assigned t
 
 It's helpful to think of rank as determining the position of each policy assignment in an ordered list, from highest rank to lowest rank. In fact, rank can be specified as any number, but these are converted into sequential values 1, 2, 3, etc. with 1 being the highest rank. When assigning a policy to a group, set the rank to be the position in the list where you want the new group policy assignment to be. If a rank is not specified, the policy assignment will be given the lowest rank, corresponding to the end of the list.
 
-Group policy assignment allows you to easily manage policies across different subsets of users within your organization. Group policy assignment is recommended for groups of up to 50000 users, but it will also work with larger groups. Group policy assignments are only propagated to users that are direct members of the group; the assignments are not propagated to members of nested groups.
+Group policy assignment allows you to easily manage policies across different subsets of users within your organization. Group policy assignment is recommended for groups of up to 50000 users, but it will also work with larger groups. 
+
+> [!Note]
```
Comment (tseward) [inline teams/teams-ps/teams/New-CsGroupPolicyAssignment.md:32]: Remove ![Note] as it cannot be parsed in PowerShell. Replace with a standard paragraph.

**Comment by get-itips** [comment]
Agree when @tseward suggestion is applied.

**Comment by JSHooper** [comment]
Also agreed, I had forgotten this gets parsed in PowerShell 👍

**Comment by get-itips** [comment]
> Also agreed, I had forgotten this gets parsed in PowerShell 👍

Can you kindly apply the suggestion? Thanks

**Comment by get-itips** [comment]
@JSHooper James, can you kindly apply the suggestions? Thanks

**Comment by yogkumgit** [comment]
@JSHooper  please implement the suggested changes so that we can proceed to merge this PR. Thanks

**Comment by get-itips** [comment]
Hello @JSHooper James, suggestions are still not applied, can you kindly do it? thank you!

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**